doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1545,19 +1545,25 @@ whose general syntax is
    :name: =>
 
 .. prodn::
-   i_item ::= @i_pattern %| @s_item %| @clear_switch %| {? %{%} } /@term
+   i_item ::= @i_pattern %| @s_item %| @clear_switch %| @i_view %| @i_block
 
 .. prodn::
-   s_item ::= /= %| // %| //=
+   i_view ::= {? %{%} } /@term %| /ltac:( @tactic )
 
 .. prodn::
-   i_pattern ::= @ident %| _ %| ? %| * %| {? @occ_switch } -> %| {? @occ_switch }<- %| [ {?| @i_item } ] %| - %| [: {+ @ident } ]
+   s_item ::= /= %| // %| //= %| /@num= %| /@num/ %| /@num/@num=
 
-The ``=>`` tactical first executes tactic, then the :token:`i_item` s,
+.. prodn::
+   i_block ::= [^ @ident ] %| [^~ @ident ] %| (^ @ident ) %| (^~ @ident )
+
+.. prodn::
+   i_pattern ::= @ident %| > @ident %| _ %| ? %| * %| + %| {? @occ_switch } -> %| {? @occ_switch }<- %| [ {?| @i_item } ] %| ( {?| @i_item } ) %| - %| [: {+ @ident } ]
+
+The ``=>`` tactical first executes :token:`tactic`, then the :token:`i_item` s,
```

Comment:
`i_view` and `i_block` are not mentioned in this paragraph. It would be good if that was the case. It would also be good if the order of the description could be the same as in the grammar (easier to follow). You can also change the order of the grammar.

Reply:
done